content/post/web-analytics-process.md

```diff
@@ -0,0 +1,83 @@
+---
+author: Ana Kravitz
+title: "Web Analytics Maintenance Process"
+publishDate: "2018-07-28"
+draft: true
+tags: 
+ - web analytics
+ - tracking plan
+ - event tracking
+---
+
+A web analytics implementation project often starts with quite a lot of fanfare and resources. There will usually be an audit and needs assessment process to determine what tracking needs to be implemented or fixed, an implementation project plan identifying task owners and dates, and earmarked hours from the development team for tasks like implementing tracking code and building a data layer. All of this generally ensures that there is satisfactorily comprehensive and accurate tracking in place at the end of the project. So why do we still regularly see web analytics issues?
+<!--more-->
+
+Common web analytics issues:
+* Broken tracking
+* Unexplained data discrepancies
+* Untagged marketing campaigns or site elements
+* Confusing and messy naming conventions
+* Etc…
+
+## Why web analytics breaks
```

Comment:
should this be `break` instead of `breaks`? (I'm honestly not sure which is grammatically correct, but this sounded a bit off to my ear.

Reply:
It sounds right to me, since it's singular. For example, we'd say "web analytics is the analysis of web data", not "web analytics are..." I'll switch it to "web analytics tracking" though in case it sounds weird to anyone else.